index.js

```diff
@@ -52,6 +52,7 @@ function requireAllModels(Implementation, modelsDir, displayMessage) {
 
 exports.Schemas = Schemas;
 exports.logger = logger;
+exports.ResourcesRoute = {};
```

Comment:
`ResourcesRoutes` with a `s` seems more understandable.

Reply:
`RoutesPerModel` is a better name?
`RoutesPerModel.order.list`?

Reply:
I agree about RoutesPerModel is more explicit. But I also want to be consistent with the other classes: ResourceSerializer, ResourceDeserializer.

If we change, let's schedule a global name migration with backward compatibility. Honestly, I think we can move forward like this. Tell me what you think.